.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+  ignore:
+  - dependency-name: defusedxml
+    versions:
+    - 0.7.0
+  - dependency-name: django
+    versions:
+    - 3.1.5
+    - 3.1.6
+  - dependency-name: django-markdownify
+    versions:
+    - 0.8.2
```

Comment:
Remove these ignored versions so that dependabot checks every version again. This possibly leads to duplicate PRs, but no version is left out.
```suggestion
```